app/controllers/hosts_controller.rb

```diff
@@ -684,11 +684,11 @@ def statuses
     render :json => statuses
   end
 
-  def hosts_path(*args)
+  def current_hosts_path(*args)
```

Comment:
Don't controllers include all helpers? So isn't this duplicating the `current_hosts_path` from `ApplicationHelper`?

Reply:
Updated it use `helpers.current_hosts_path`  instead